scripts/certs-generate.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
```

Comment:
rename as generate-test-certs?

Reply:
"test" is not that specific. "self-signed" would be more descriptive. I lead with "certs-" so it is easier for someone to find that is looking for certs. I am okay with changing the name to whatever the reviewers like though.